baybe/surrogates/gaussian_process/core.py

```diff
@@ -108,14 +110,23 @@ def to_botorch(self) -> Model:  # noqa: D102
 
         return self._model
 
+    @staticmethod
+    def _make_parameter_scaler(
+        parameter: Parameter,
+    ) -> ScalerProtocol | Literal["passthrough"]:
+        # See base class.
+
+        # For GPs, we use botorch's built-in machinery for scaling
```

Comment:
a bit surprised, why? is this better in any way? it certianly mixes up things which is not great. so let me know why you opten for that

Reply:
Mostly for historical reasons, because that's how it used to be (i.e. never touch a running system). I guess I could also change it ... 🤔  I mean, the botorch stuff has a lot more functionality under the hood, but I hope 🤞🏼 that we don't rely on any of it without knowing. Can try and see what happens

Reply:
id prefer mixing stuff only if theres a benefit, so far it doesnt sound like it and is just historical

Reply:
Changed